jobs/shareMethod.groovy

```diff
@@ -41,7 +41,9 @@ def getLockedResourceName(resources,label_name){
 
 def buildAndPublish(){
     stage("Packages Build"){
-        load("jobs/build_debian/build_debian.groovy")
+        retry(3){
```

Comment:
1. 3 exist in multiple places, please assigned '3' to a variable name to make it more meaningful
2. also please add a annotation about why 3, or just 'it's a experience value'